src/extension-metadata/__test__/data.spec.js

```diff
@@ -0,0 +1,117 @@
+import { indexAdded, indexRemoved, min, getDescription } from '../data';
```

Comment:
Is all of this code new?

Reply:
No, they are not.

Reply:
I mean this line of code is new. But those methods are not.